### DISCUSSION TITLE
Rule-based Matching Guides: on_match = add_event_ent NOT working

### DISCUSSION THREAD

**Discussion by nick-sh-oh** [explosion/spaCy#12867] [Help: Other Questions]
I am following the guide, but the demo function for Google I/O does not seem to work... `on_match` works fine with other patterns, but is this a problem when trying to identify `entity`? 

## Which page or section is this issue related to?

https://spacy.io/usage/rule-based-matching#on_match

**Comment by shadeMe** [comment]
The example on the website works as expected and returns the following output:
```
Google I/O
[(3492015959717556363, 5, 9)]
```

Could you post your code and describe the problem that you're having in more detail?
